### PR TITLE
Add twitter website tag

### PR DESCRIPTION
**Pull request by davidebukali**
#1151

### CODE REVIEW

File: di_website/templates/layouts/default.html [hunk 1]
```diff
@@ -10,6 +10,16 @@
...
+  <!-- Twitter universal website tag code -->
+  <script>
+  !function(e,t,n,s,u,a){e.twq||(s=e.twq=function(){s.exe?s.exe.apply(s,arguments):s.queue.push(arguments);
+  },s.version='1.1',s.queue=[],u=t.createElement(n),u.async=!0,u.src='//static.ads-twitter.com/uwt.js',
+  a=t.getElementsByTagName(n)[0],a.parentNode.insertBefore(u,a))}(window,document,'script');
+  // Insert Twitter Pixel ID and Standard Event data below
+  twq('init','o8fck');
+  twq('track','PageView');
+  </script>
+  <!-- End Twitter universal website tag code -->
```
Comment (edwinmp) [inline di_website/templates/layouts/default.html:22]: Why is this required if you're using Google Tag Manager? The whole point of GTM is to avoid this kind of thing ...

Comment (davidebukali) [reply di_website/templates/layouts/default.html:22]: Closed this as the tag fires without it included.
